boards/configurations/microbit_v2/microbit_v2-test-dynamic-app-load/src/io.rs

```diff
@@ -76,7 +72,7 @@ pub unsafe fn panic_fmt(pi: &PanicInfo) -> ! {
 
     // MicroBit v2 has a microphone LED, use it for panic
 
-    use core::ptr::{addr_of, addr_of_mut};
+    use core::ptr::addr_of_mut;
```

Comment:
I see a few changes in this PR that look like auto-formatter cleanup of imports, is rust-analyzer doing this now? Just curious